disorder-core/disorder-core.cabal

```diff
@@ -17,6 +17,7 @@ library
                      , text                            >= 1.1        && < 1.3
                      , process                         == 1.2.*
                      , QuickCheck                      == 2.7.*
+                     , ieee754                         == 0.7.*
```

Comment:
Maybe not judging by the [existing versions](https://hackage.haskell.org/package/ieee754).